walletunlocker/service_test.go

```diff
@@ -374,6 +402,11 @@ func TestUnlockWallet(t *testing.T) {
 				"got %d", testRecoveryWindow,
 				unlockMsg.RecoveryWindow)
 		}
+
+		// We'll now close the done channel to unlock the service to be
+		// able to accept another request.
+		close(unlockMsg.Done)
```

Comment:
Why do we need to close the done chan in tests that don't make any other calls, just for the sake of mimicking reality in case we add more to the test?

Reply:
Just to try to make it as close to reality as possible IMO.